Makefile

```diff
@@ -3,6 +3,7 @@ SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
 ROOT := $(shell pwd)
 LOCAL_ENI_PLUGIN_BINARY=bin/plugins/eni
 LOCAL_IPAM_PLUGIN_BINARY=bin/plugins/ipam
+LOCAL_BRIDGE_PLUGIN_BINARY=bin/plugins/bridge
```

Comment:
That's happening in a separate PR. Being tracked her https://github.com/aws/amazon-ecs-cni-plugins/issues/38